packages/components/notifications/src/content-notification/index.js

```diff
@@ -1 +1,3 @@
 export { default } from './content-notification';
+
+export { default as version } from './version';
```

Comment:
The version should be part of the `notifications` package, not the single component.

Reply:
The version should go into the `src` folder. I pushed a commit: https://github.com/commercetools/ui-kit/pull/1451/commits/dd8a0d44da8f116d632a2c1ade0ff1f40f9861cc